libadikted/msg_log.c

```diff
@@ -26,6 +26,50 @@ short message_hold;
 unsigned int message_getcount;
 char *msgout_fname;
 
+/**
+ * Global state prevents massive fopen()/fclose()
+ */
+#define ALLOW_GLOBAL_MSGOUT_FP  1
```

Comment:
Consider adding an explanation to the comment above, as to what setting the value either 1 or 0 does - alternatively you could rename `ALLOW_GLOBAL_MSGOUT_FP` to something like `REUSE_MESSAGE_LOG_FILE_HANDLE` which in my personal opinion is a bit more self documenting. This is entirely optional, just a personal taste of mine.

Reply:
Done